Makefile

```diff
@@ -0,0 +1,150 @@
+  volume or image.
+
+  - stop              Stops the Madara node
+
+  [ RESTARTING MADARA ]
+
+  Restarts Madara, possibly cleaning containers, images and volumes in the
+  process. Note that it is also required for you to have added the necessary
+  secrets to `./secrets/`, or the nodes will fail to restart.
+
+  - restart           Restarts the Madara node
+  - frestart          Perform a full clean and restarts the Madara node
+
+  [ LOGGING MADARA ]
+
+  This will show logging outputs for the Madara container. Defaults to following
+  the output, <Ctrl-C> to quit.
+
+  - logs              View logs for Madara
+
+  [ DOWLOAD DEPENDENCIES ]
+
+  Images are downloaded from the github container registry. Note that to avoid
+  continuousy downloading images those are exported to a `tar.gz` as artefacts.
+
+  - images             Downloads the Madara Docker image
+
+  [ CLEANING DEPENDECIES ]
+
+  Will remove running containers, images and even local db. Use the latter with
+  care as removing the local db will force a resync from genesys.
+
+  - clean              Stop containers and prune images
+  - clean-db           Perform clean and remove local database
+  - fclean             Perform clean-db and remove local images
+
+  [ OTHER COMMANDS ]
+
+  - help               Show this help message
+
+endef
+export HELP
+
+SECRETS  := .secrets/rpc_api.secret
+
+# dim white italic
+DIM      := \033[2;3;37m
+
+# bold cyan
+INFO     := \033[1;36m
+
+# bold green
+PASS     := \033[1;32m
+
+# bold red
+WARN     := \033[1;31m
+
+RESET    := \033[0m
+
+.PHONY: all
+all: help
+
+.PHONY: help
+help:
+	@echo "$$HELP"
+
+.PHONY: start
+start: images $(SECRETS)
+	@echo -e "$(DIM)running$(RESET) $(PASS)madara$(RESET)"
+	@docker compose -f compose.yaml up -d
+
+.PHONY: stop
+stop:
+	@echo -e "$(DIM)stopping$(RESET) $(WARN)madara$(RESET)"
+	@docker compose -f compose.yaml stop
+
+.PHONY: logs
+logs:
+	@echo -e "$(DIM)logs for$(RESET) $(INFO)madara$(RESET)";
+	@docker compose -f compose.yaml logs -f -n 100;
+
+.PHONY: images
+images: image.tar.gz
+
+image.tar.gz:
+	@echo -e "$(DIM)downloading$(RESET) $(PASS)madara$(RESET)"
+	@docker pull ghcr.io/madara-alliance/madara:latest
+	@docker tag ghcr.io/madara-alliance/madara:latest madara:latest
+	@docker rmi ghcr.io/madara-alliance/madara:latest
+	@docker image save -o image.tar.gz madara:latest
+
+.PHONY: clean
+clean: stop
+	@echo -e "$(DIM)pruning containers$(RESET)"
+	@docker container prune -f
+	@echo -e "$(DIM)pruning images$(RESET)"
+	@docker image prune -f
+	@echo -e "$(WARN)images cleaned$(RESET)"
+
+.PHONY: clean-db
+clean-db:
+	@echo -e "$(WARN)This action will result in irrecoverable loss of data!$(RESET)"
+	@echo -e "$(DIM)Are you sure you want to proceed?$(RESET) $(PASS)[y/N] $(RESET)" && \
+	read ans && \
+	case "$$ans" in \
+		[yY]*) true;; \
+		*) false;; \
+	esac
+	@make --silent clean
+	@echo -e "$(DIM)removing madara database on host$(RESET)"
+	@sudo rm -rf /var/lib/madara;
```

Comment:
and db path under DB_PATH := /var/lib/madara

Reply:
For many reasons I feel that elevating privilages in a makefile are bad practice: user loses control, some environments may not require it (container), it can break processes like automated integrations, some of your makefile rules dont even require sudo, etc wdyt?

Reply:
This has been fixed. Remove `sudo` from the Makefile.